### PR TITLE
FINERACT-1956: Added the skeleton structure necessary to publish doc …

### PR DESCRIPTION
**Pull request by jogathogo**
## Description
Added the necessary folders and files for Antora to build the documentation site from asciidoc i.e 
Refer to https://docs.antora.org/antora/latest/

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Comment by jogathogo** [comment]
@vidakovic please review and advise.

**Comment by jdailey** [comment]
Please reopen

File: fineract-doc/modules/contributor/pages/key-design-principles.adoc
Comment (jdailey) [inline fineract-doc/modules/contributor/pages/key-design-principles.adoc:1]: This needs to be trimmed down.  The current versions of building blocks should be listed in the Readme with each release.

**Review by jdailey — COMMENTED** [review]
Would be good to accept this with some minor changes.   I'd like to remove anything that is updated frequently, like versions of underlying technologies.  Those belong elsewhere.   

I think the section on CQRS also needs updating... it was never fully implemented across the application and there have been changes to the strategy in the past few years.

**Comment by jogathogo** [comment]
The suggested bits have been removed. Review and advise.

**Comment by jdailey** [comment]
LGTM

**Comment by vidakovic** [comment]
Please don't merge this one before we've created release 1.9.0. Let's target 1.10.0. Jira tickets updated.

**Comment by jdailey** [comment]
Aleks - can we push now?

**Comment by jdailey** [comment]
No conflicts w base branch.

**Comment by jogathogo** [comment]
> Please don't merge this one before we've created release 1.9.0. Let's target 1.10.0. Jira tickets updated.

@vidakovic now that 1.9 is out, can we resume this? This PR is purely documentation related and does not contain any fineract code and therefore should be safe to merge without breaking anything.

**Review by Sanyam96 — COMMENTED** [review]
LGTM

**Comment by adamsaghy** [comment]
@jogathogo Please squash your commits and check why the tests were failing!

**Comment by adamsaghy** [comment]
@jdailey @vidakovic @jogathogo Should we reopen and merge this one?

**Pull request closed without merging**